rpc/src/v1/types/sync.rs

```diff
@@ -240,9 +240,11 @@ mod tests {
 	fn test_serialize_transaction_stats() {
 		let stats = TransactionStats {
 			first_seen: 100,
-			propagated_to: map![
-				H512::from_low_u64_be(10) => 50
-			],
+			propagated_to: {
+				let mut map = BTreeMap::new();
```

Comment:
why not a `btreemap!`?

Reply:
nit, I think I started to investigate if we could get rid of `map` completely but forgot this.

Reply:
I thought you left it on purpose. Why use a macro when it doesn’t save much typing and doesn’t make the code clearer?

Reply:
I was just too lazy,  my rule of thumb is when we have more than 2-3 items in a BTreeMap then I use `btreemap!` but then we should clean up the entire crate.

At least after this change we are consistent in the crate but I saw that we can easily remove `maplit` in the `rpc crate`